### PR TITLE
Twitter Upload Media Action bug fix

### PR DESCRIPTION
**Pull request by michelle0927**
Error "SegmentIndex must be between 0 and 999." observed occurring for 4MB files. This PR increases the segment size to reduce the number of segments to less than 999 and prevent the error from occurring.

See #5066

### CODE REVIEW

File: components/twitter/actions/upload-media/upload-media.mjs [hunk 1]
```diff
@@ -41,7 +41,7 @@ export default {
     });
 
     // APPEND
-    const size = 5012;
+    const size = 8000;
```
Comment (andrewjschuang) [inline components/twitter/actions/upload-media/upload-media.mjs:44]: As per [the docs](https://developer.twitter.com/en/docs/tutorials/uploading-media) the max chunk size is 1MB, should we use this value?

```suggestion
    const size = 1000000;
```

Comment (michelle0927) [reply components/twitter/actions/upload-media/upload-media.mjs:44]: @andrewjschuang I found that setting the value higher resulted in an error saying that the request headers were too large.

Comment (andrewjschuang) [reply components/twitter/actions/upload-media/upload-media.mjs:44]: Higher than 8000?

**Review by andrewjschuang — DISMISSED** [review]

**Review by andrewjschuang — APPROVED** [review]
LGTM!

**Comment by Hassan-Abbas-SQA** [comment]
TEST PASSED
200 Seconds 4.5 mb mp4 video
image

**Comment by gabriel-kaam** [comment]
Hello @michelle0927 , me again 🥲

I have the same issue with this file

Capture d’écran 1657

But this time , the file can be properly uploaded via the Twitter UI , so the format is correct .

Video is 14.7mo , fresh out of a CloudConvert conversion .

Here's the MP4 file is you want to try: https://transfer.sh/tvnYmC/7E4267863858A5842A69CA17A1D7ED92_video_dashinit.mp4 

I feel bad for disturbing you with my problems ; is there a way for me to enable a Debug mode on my Workflow to get more insights on what's going wrong ?

**Comment by michelle0927** [comment]
@gabriel-kaam I'm so sorry you keep seeing these errors. I've created one more PR (https://github.com/PipedreamHQ/pipedream/pull/5445) that should fix the issue. After updating, I was able to successfully upload and create a tweet with the .mp4 video you linked to above.

**Pull request merged**